content/configuration/dev-server.md

```diff
@@ -478,3 +478,8 @@ watchOptions: {
 If this is too heavy on the file system, you can change this to an integer to set the interval in milliseconds.
 
 See [WatchOptions](/configuration/watch) for more options.
+
+
+## `devServer.progress` - CLI only
+
+Output running progress to console.
```

Comment:
In https://github.com/webpack/webpack.js.org/pull/997/files you can click the edit icon to make a second commit:
![2017-03-08-12-16-23-001](https://cloud.githubusercontent.com/assets/782446/23701887/16db7738-03f9-11e7-8d31-103033fb2314.png)